### PR TITLE
CompleteTask In Notification

### PR DESCRIPTION
**Pull request by jeppekikt**
and fixed bug with sound in luncher app

### CODE REVIEW

File: TaskSharper.WPF.Common/Components/Notification/NotificationViewModel.cs [hunk 1]
```diff
@@ -130,54 +160,81 @@ private Task ShowNotification(Events.Resources.Notification notification)
...
-                notificationSound.Play();
+                try
+                {
+                    notificationSound.Load();
+                    notificationSound.Play();
+                }
+                catch (Exception e)
+                {
+                    Console.WriteLine(e);
+                    throw;
```
Comment (madsmarcussen) [inline TaskSharper.WPF.Common/Components/Notification/NotificationViewModel.cs]: Handle this exception properly - rethrowing an exception like this is the same as not doing try-catch at all.

**Review by madsmarcussen — CHANGES_REQUESTED** [review]
Fix error handling please :)

File: TaskSharper.WPF.Common/Components/Notification/NotificationViewModel.cs [hunk 2]
```diff
@@ -115,10 +127,28 @@ private void SetScrollButtonsVisibility(EventResources.ScrollButtonsEnum state)
             }
             else
             {
-                await ShowNotification(notification);
+                if (notification.Event.Type == EventType.Task && notification.Event.MarkedAsDone)
```
Comment (alminde7) [inline TaskSharper.WPF.Common/Components/Notification/NotificationViewModel.cs]: Vend denne if/else.. der bliver alligevel ikke gjort noget i if :)

File: TaskSharper.WPF.Common/Components/Notification/NotificationViewModel.cs (hunk 1 above)
Comment (jeppekikt) [reply TaskSharper.WPF.Common/Components/Notification/NotificationViewModel.cs]: ops

**Review by madsmarcussen — APPROVED** [review]

File: TaskSharper.WPF.Common/Components/Notification/NotificationViewModel.cs (hunk 2 above)
Comment (jeppekikt) [reply TaskSharper.WPF.Common/Components/Notification/NotificationViewModel.cs]: fixed

File: TaskSharper.WPF.Common/Components/Notification/NotificationViewModel.cs (hunk 1 above)
Comment (jeppekikt) [reply TaskSharper.WPF.Common/Components/Notification/NotificationViewModel.cs]: fixed and logging it

**Review by alminde7 — APPROVED** [review]

**Pull request merged**